lesson17/index.js

```diff
@@ -0,0 +1,114 @@
+"use strict";
+
+const resourceUrl = "https://jsondata.okiba.me/v1/json/9gjw3210424132243";
+const body = document.getElementById("body");
+const wrapper = document.querySelector(".wrapper");
+const box = document.querySelector(".box");
+const prevButton = document.querySelector(".prevNext__prev");
+const nextButton = document.querySelector(".prevNext__next");
+const navigationNum = document.querySelector(".navigation__num");
+const fragment = document.createDocumentFragment();
+let li;
+let imageLength;
+let currentNum = 0;
```

Comment:
```suggestion

const slides = {
  length: null
  currentNumber: 0
}
```
こう書くと何関連の変数を管理しているのかがわかりやすくなります

Reply:
基本的なことだと思いますが、オブジェクトで管理するのは盲点でした。勉強になりますm(__)m
修正しました 5e8aaf7